### PR TITLE
Tweak kubeadm certs advice

### PR DESCRIPTION
**Pull request by sftim**
For https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/, use fewer note and warning blocks, to keep the important messages focused.

Here's a [preview](https://deploy-preview-48110--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/). The only warning (strongest callout, red) that I've kept is within https://k8s.io/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/#kubeconfig-additional-users

Also, use one consistent phrase about the advice around upgrading a cluster. That means updating three pages total.

/sig cluster-lifecycle
/language en

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 1]
```diff
@@ -158,26 +170,24 @@ if you don't have specific requirements on certificate renewal and perform Kuber
 upgrades regularly (less than 1 year in between each upgrade), kubeadm will take care of keeping
 your cluster up to date and reasonably secure.
 
-{{< note >}}
-It is a best practice to upgrade your cluster frequently in order to stay secure.
-{{< /note >}}
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md:163]: Moved to page intro

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 2]
```diff
@@ -188,33 +198,25 @@ The kubelet will terminate the Pod if it's no longer in the manifest directory.
 You can then move the file back and after another `fileCheckFrequency` period, the kubelet will recreate
 the Pod and the certificate renewal for the component can complete.
 
-{{< warning >}}
-If you are running an HA cluster, this command needs to be executed on all the control-plane nodes.
-{{< /warning >}}
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md:193]: Moved into the shell snippet and also to the start of this section.

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 3]
```diff
@@ -188,33 +198,25 @@ The kubelet will terminate the Pod if it's no longer in the manifest directory.
...
-
-{{< note >}}
-`certs renew` uses the existing certificates as the authoritative source for attributes (Common
-Name, Organization, SAN, etc.) instead of the `kubeadm-config` ConfigMap. It is strongly recommended
-to keep them both in sync.
-{{< /note >}}
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md:199]: Moved earlier.

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 4]
```diff
@@ -158,26 +170,24 @@ if you don't have specific requirements on certificate renewal and perform Kuber
...
-
 If you have more complex requirements for certificate renewal, you can opt out from the default
 behavior by passing `--certificate-renewal=false` to `kubeadm upgrade apply` or to `kubeadm
 upgrade node`.
 
-{{< warning >}}
-Prior to kubeadm version 1.17 there is a [bug](https://github.com/kubernetes/kubeadm/issues/1818)
-where the default value for `--certificate-renewal` is `false` for the `kubeadm upgrade node`
-command. In that case, you should explicitly set `--certificate-renewal=true`.
-{{< /warning >}}
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md:173]: Dropped

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 5]
```diff
@@ -158,26 +170,24 @@ if you don't have specific requirements on certificate renewal and perform Kuber
...
+with the appropriate command line options. If you are running cluster with a replicated control
+plane, this command needs to be executed on all the control-plane nodes.
 
 This command performs the renewal using CA (or front-proxy-CA) certificate and key stored in `/etc/kubernetes/pki`.
 
+`kubeadm certs renew` uses the existing certificates as the authoritative source for attributes
+(Common Name, Organization, subject alternative name) and does not rely on the `kubeadm-config`
+ConfigMap.
+Even so, the Kubernetes project recommends keeping the served certificate and the associated
+values in that ConfigMap synchronized, to avoid any risk of confusion.
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md:189]: Moved out of an existing note.

**Review by dipesh-rawat — COMMENTED** [review]
Thanks!

/approve

**Pull request merged**